src/devices/BrickPi3/Models/SpiMessages.cs

```diff
@@ -11,78 +11,115 @@ public enum SpiMessageType : byte
     {
         /// <summary>None</summary>
         None = 0,
+
         /// <summary>Get manufacturer</summary>
         GetManufacturer,
+
         /// <summary>Get name</summary>
         GetName,
+
         /// <summary>Get hardware version</summary>
         GetHardwareVersion,
+
         /// <summary>Get firmware version</summary>
         GetFirmwareVersion,
+
         /// <summary>Get identifier</summary>
         GetId,
+
         /// <summary>Set LED</summary>
         SetLed,
+
         /// <summary>Get Voltage (3.3V)</summary>
         GetVoltage3V3,
+
         /// <summary>Get Voltage (5V)</summary>
         GetVoltage5V,
+
         /// <summary>Get Voltage (9V)</summary>
         GetVoltage9V,
+
         /// <summary>Get Voltage (Vcc)</summary>
         GetVoltageVcc,
+
         /// <summary>Set address</summary>
         SetAddress,
+
         /// <summary>Get sensor type</summary>
         SetSensorType,
+
         /// <summary>Get sensor 1</summary>
         GetSensor1,
+
         /// <summary>Get sensor 2</summary>
         GetSensor2,
+
         /// <summary>Get sensor 3</summary>
         GetSensor3,
+
         /// <summary>Get sensor 4</summary>
         GetSensor4,
+
         /// <summary>I2C transaction 1</summary>
         I2CTransact1,
+
         /// <summary>I2C transaction 2</summary>
         I2CTransact2,
+
         /// <summary>I2C transaction 3</summary>
         I2CTransact3,
+
         /// <summary>I2C transaction 4</summary>
         I2CTransact4,
+
         /// <summary>Set motor power</summary>
         SetMotorPower,
+
         /// <summary>Set motor position</summary>
         SetMotorPosition,
+
         /// <summary>Set proportional factor (KP) of motor position controller</summary>
         SetMotorPositionKP,
+
         /// <summary>Set derivative factor (KD) of motor position controller</summary>
         SetMotorPositionKD,
+
         /// <summary>Set motor angular speed (degrees per second)</summary>
-        setMotorDps,
+        SetMotorDps,
```

Comment:
All of these casing changes are breaking changes. That said I think it's ok to take them as it was a mistake to not catch this in the beginning, but it is something we will probably have to document for vNext so we should just keep that in mind.

Reply:
Yea, guess we should just make a note in the release docs. Do we keep them (or some kind of changelog for the end user) somewhere in the repo?

Reply:
@pgrawehr, no worries, @joperezr knows how to find all breaking changes between 1.0 and current version with ApiCompat tool. We can use that when writing release notes, no action on your end needed I believe

Reply:
Yeah unfortunately we don't have a process established yet for breaking changes, but we can use the ApiCompat tool to find out all of these and we can just call them out on the vNext release notes.